packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -220,7 +220,7 @@ const ActivityDetailsSidebar = (props: Props) => {
             />
           )}
 
-          {selectedTeam.tier === 'starter' && !selectedTemplate?.isFree ? (
+          {selectedTeam.tier === 'starter' && selectedTemplate && selectedTemplate.isFree ? (
```

Comment:
-1 Fails, in a starter team I can see premium templates with a "Start activity" button. When starting it, it seems to work but the server prints `SEND TO SENTRY Error: User does not have access to this premium template`.
On master I see the upgrade button in this case. Tested with Team Charter and SWOT Analysis

Reply:
I'm sorry, this is just a mistake from testing phase, I removed `!` from `!selectedTemplate.isFree` condition. could you double check now?

Reply:
Should have seen this :facepalm:

Reply:
nah, that should not have happened! thank you for actually checking that.